apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -124,6 +124,20 @@ static void startAppCenter(Application application, String startTypeString) {
                 Toast.makeText(application, "No Update Track API in this build", Toast.LENGTH_SHORT).show();
             }
         }
+        boolean automaticCheckForUpdate = sSharedPreferences.getBoolean(application.getString(R.string.appcenter_distribute_disable_check_for_update_key), false);
```

Comment:
```suggestion
        boolean automaticCheckForUpdate = sSharedPreferences.getBoolean(application.getString(R.string.appcenter_distribute_disable_check_for_update_key), true);
```
To fix:
<img width="1224" alt="Screen Shot 2020-02-20 at 5 08 39 PM" src="https://user-images.githubusercontent.com/623672/74994810-e9100e00-5403-11ea-894c-9aa1d504fb6f.png">